### DISCUSSION TITLE
Is it Allowed to Host NFT minting DApp to GitHub Pages?

### DISCUSSION THREAD

**Discussion by warengonzaga** [community/community#23077] [Other Feature Feedback, Questions, & Ideas]
<p>Just wondering if it is allowed to host NFT minting DApp with GitHub Pages? This is a complex topic that I want to tackle here and need help finalizing this. Basically, is it allowed to host Web3-related pages to the GitHub pages including minting pages where users are able to mint to a decentralized app that is currently hosting via GitHub pages? Any answer would be very much appreciated! :fire:</p>

**Comment by yoannchaudet** [comment]
<p>Hello and welcome here,</p>
<p>Technically speaking if your “NFT minting DApp” is a pure static web app that can work with the HTTP headers Pages returns, it may be hosted on Pages.</p>
<p>Legally, if you are planning on making any money with such an app (fiat or not), you are probably violating the <a href="https://docs.github.com/en/site-policy/github-terms/github-terms-for-additional-products-and-features#pages">terms of services</a>.</p>
<p>I suggest you reach out to the support in order to get a better answer.</p>

**Comment by warengonzaga** [comment]
<p>hey, <a class="mention" href="/yoannchaudet">@yoannchaudet</a> thank you for getting back to me, how to get in touch with the team directly? Technically, I was able to make it work with the GH pages the only problem in my end is that is it legal to host minting dapp in your hosting which is beneficial to know for the rest of the Web3 community. I’m working at thirdweb :fire:</p>

**Comment by warengonzaga** [comment]
<p>Update: contacted the team via support ticket. Will update everyone here once I receive a response.</p>

**Reply by o-tsaruk** [reply]
hey, did you get the answer?=)

**Reply by warengonzaga** [reply]
Unfortunately, they didn't answer it directly. It is clear you can't host minting pages to GitHub.
